MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1290,6 +1305,7 @@
 				E2C1CDB2265E439300E158E0 /* MapboxMobileEvents.xcframework in Frameworks */,
 				E2C1CDB5265E43D500E158E0 /* Quick.xcframework in Frameworks */,
 				E2C1CDB7265E43E100E158E0 /* Nimble.xcframework in Frameworks */,
+				2C67751C290AC4A6009C5EE4 /* MapboxNavigationNative.xcframework in Frameworks */,
```

Comment:
Nit: is this one really needed? Just want to make sure we do not add any excessive dependency.

Reply:
This new one is needed for test for my new `NativeNavigatorSpy` in TestHelper. It adds a dependency on `NavigationNative` for TestHelper.
Without it we got build error such as 
> error build: Undefined symbol: _OBJC_CLASS_$_MBNNNavigator